src/platform/forms/constants.js

```diff
@@ -238,6 +238,8 @@ export const TRACKING_PREFIXES = {
   [VA_FORM_IDS.FORM_20_0996]: 'decision-reviews-va20-0996-',
   [VA_FORM_IDS.FORM_VA_2346A]: 'bam-2346a-',
   [VA_FORM_IDS.FORM_5655]: 'fsr-5655-',
+  [VA_FORM_IDS.FORM_28_8832]: '28-8832-',
+  [VA_FORM_IDS.FORM_26_1880]: '26-1880-',
 };
```

Comment:
@allisonlu @adamwhitlock1 do you know if these prefixes need to match something defined elsewhere?

Reply:
I don't think they need to match anything specifically, but are usually the same string used to get the app url with an extra - added basically `getAppUrlImpl('hca')`

Reply:
gotcha thanks :) yea looked like there wasn't a well defined pattern for these atm across the board, and still a little confused if we need these for anything other than our `isSIPEnabledForm` helper func 🤔 .  If we can confim this is the only use-case for these constants, maybe we can refactor here a bit to remove the consts, or autogenerate using configured values